contributor/03-contributor-ethos.md

```diff
@@ -0,0 +1,64 @@
+# Draft comments
+
+ Outline for this section
+
+     Contributors by definition work in teams that they are not familiar with. This section explains how to avoid common pitfalls when crossing team boundaries.
+
+ ## Article
+
+In the last segment we have motivated why you would want to reuse components and become active as a Contributor. This artice is sharing best practices on how to successfully contribute your changes to the host team's code base.
+
+ ### Contributor Ethos
+
+An InnerSource Contributor trying to make a contribution to the host team essentially is a guest in their home. In general a good guest is expected to behave in a certain way:
+
+* They knock at the door.
+* They anticipate and follow house rules.
+* They understand they are not the home owner and acty accordingly.
+
+How do these expectations apply to InnerSource projects?
+
+#### Getting in
+
+In the real world their are several ways that guests are invited in:
+
+* An "open" sign lets them know they can walk right in without knocking.
+* Knock on the door unexpected.
+* Knock on the door only when invited.
+
+With InnerSource projects there are several ways that code gets in:
+
+Sometimes you just open a pull request. At times you may need to get a feature request accepted before opening the pull request that fulfills it.
+
+In some organisations you may need to talk to the host team in person in addition to interacting with the project digitally. 
+
+In the same way that real-life hosts don't always appreciate a stranger knocking on their door, some InnerSource hosts appreciate some heads-up before seeing a pull request gets opened to their repo. Typically there will be an issue tracker where new features are discussed before being submitted as a pull request. A useful alternative are archived, searchable, linkable but unstructured discussion platforms: Thank of mailing lists, chat plattforms or online fora.
+
+In all of these interactions, be prepared to "sell" your contribution to the host team. Articulate the value that the contribution will bring to their ecosystem.
+
+The host team will be the one taking over maintenance for your changes. It makes sense to offer to fulfill a 30-day warranty on your submission. This can alleviate the host teams' fear of the contributors not being available for support with fixing bugs after the time on contribution.
+
+Many InnerSource projects outline how they like to be approached by potential Contributors in their README.md or CONTRIBUTING.md files. Following those recommendations greatly increases your chances of getting your contribution accepted.
```

Comment:
I know you're just following the order in the outline, but as I read the article now I feel like this paragraph would go better. before the paragraphs on "selling".

Reply:
In the sense of "this gives the contributing person good arguments for the sale / allows them to make a good offer"?